### DISCUSSION TITLE
Auto block breaker

### DISCUSSION THREAD

**Discussion by LoganDark** [TechReborn/TechReborn#2871] [Show and tell]
One of my friends tried TechReborn, but instantly got disappointed by the lack of a machine that can automatically break blocks. :/

**Comment by xanthian** [comment]
what.

**Reply by LoganDark** [reply]
A block breaker? Like a machine that breaks the block in front of it? As if it was mined by a player? ...

**Reply by Ayutac** [reply]
I have started something along these lines, dunno if it was the quarry or the miner. Check out my TR branches if you wanna help move it along

**Comment by xanthian** [comment]
you mean... a pickaxe?
🙄

**Reply by LoganDark** [reply]
An automatic pickaxe you dork

A pickaxe has to be held by a player and manually used to break blocks

**Reply by Ayutac** [reply]
please stay polite, everyone

**Reply by LoganDark** [reply]
> please stay polite, everyone

Should be saying that to the one who straight up said it's a "crap / lazy idea", I'm not the one being hostile in any way

Talk to Famous5000 if you need to report any activity on this issue, I'm already tired of being treated like this

**Reply by Ayutac** [reply]
I don't know what kind of communication you are used to, but when I say "everyone" I mean *everyone* ;-)

**Comment by xanthian** [comment]
sounds like a crap / lazy idea to me. 
downvoted

**Reply by Famous5000** [reply]
How is an automatic block breaker a crap/lazy idea? Have you heard of a "quarry"?

**Reply by LoganDark** [reply]
So farms are crap/lazy, automatic cobblestone generators are crap/lazy. In a tech mod. With machines and automation. WTAF

**Reply by xanthian** [reply]
theres already a quarry mod addon for TR

**Reply by Famous5000** [reply]
> theres already a quarry mod addon for TR

The block breaker **doesn't specifically need to be a quarry.** It can literally just be used to break **one block** in a controlled manner.

**Reply by SimonFlapse** [reply]
> > theres already a quarry mod addon for TR
> 
> The block breaker **doesn't specifically need to be a quarry.** It can literally just be used to break **one block** in a controlled manner.

So like the Block Breaker (and companion: Block Placer) from Industrial Foregoing?

Example: https://ftb.fandom.com/wiki/Block_Breaker_(Industrial_Foregoing)

**Reply by Famous5000** [reply]
> > > theres already a quarry mod addon for TR
> > 
> > 
> > The block breaker **doesn't specifically need to be a quarry.** It can literally just be used to break **one block** in a controlled manner.
> 
> So like the Block Breaker (and companion: Block Placer) from Industrial Foregoing?
> 
> Example: https://ftb.fandom.com/wiki/Block_Breaker_(Industrial_Foregoing)

Yeah, basically. It's a block that I was hoping would exist in this mod considering it focuses so heavily on automation.

**Reply by SimonFlapse** [reply]
> > > > theres already a quarry mod addon for TR
> > > 
> > > 
> > > The block breaker **doesn't specifically need to be a quarry.** It can literally just be used to break **one block** in a controlled manner.
> > 
> > 
> > So like the Block Breaker (and companion: Block Placer) from Industrial Foregoing?
> > Example: https://ftb.fandom.com/wiki/Block_Breaker_(Industrial_Foregoing)
> 
> Yeah, basically. It's a block that I was hoping would exist in this mod considering it focuses so heavily on automation.

I've given it a shot, and made a block breaker machine. See #2876 It will probably need some more work for it to be ready for release (And an approval)

**Comment by SimonFlapse** [comment]
The block breaker and companion machine the block placer will probably be a part of the next release of Tech Reborn.

The changes has been merged into the current 1.18 branch. (7f2912009b55d1f893c8eecc8361afacc7637323)

**Reply by LoganDark** [reply]
Thank you! Guess it's not such a horrible idea after all, eat it xanthian :P
